### ISSUE TITLE
Visual Glitch on waters

### ISSUE THREAD

**Issue by Ant-V** [Gcenx/WineskinServer#402]
**Environment (please complete the following information):**
 - OS: 
 - Winery version: 1.8.4.2
 - Master Wrapper version: Wineskin-2.9.1.9
 - Engine version: WS11WineCX64Bit22.1.1-rc2
 - Hardware: iMac M1 2021 16GB
 - Application/Game Name: Skyrim Special Edition (GoG)

**Describe the bug**
The water seems to have some weird glitch, appearing as an iridescent rainbow of colors, either not having loaded some texture or something else. 
I believe it is in the engine version, as the same issue does not exist in Crossover.

**To Reproduce**
Make new Wrapper, install downloaded installer, run game, look at water.
Faster way to see the bug is to open the map, as the same issue appears on any water-surface that is outside.

**Expected behavior**
Water appears as water in games from 10+ years ago, instead of an oil spill...

**Screenshots**
![Screenshot 2023-05-22 at 03 09 21](https://github.com/Gcenx/WineskinServer/assets/7198995/6b9c1ea1-92c9-42c1-b83a-a1c0a5af2425)
![Screenshot 2023-05-22 at 03 06 49](https://github.com/Gcenx/WineskinServer/assets/7198995/b6e53fc6-dcd7-42a2-bbf8-5eb8edb21963)
<img width="2240" alt="Screenshot 2023-05-23 at 19 28 47" src="https://github.com/Gcenx/WineskinServer/assets/7198995/03f29686-acb1-4714-a165-e9fd4a23e6f1">

Colors change depending on angle you are looking them from. 

**LastRunWine.log**
[LastRunWine.log](https://github.com/Gcenx/WineskinServer/files/11546482/LastRunWine.log)

**Additional context**

I believe it is in the engine version, as the same issue does not exist in Crossover.
I tried running a new wrapper with a different Engine, but game does not launch, only spawning minimally active wine64-preloader apps in the system monitor.
Also tried mods that change the water in different ways; nothing happened.
Any and all reflection options in the .ini do not seem to change anything.

**Comment by Gcenx** [comment]
I have a feeling this was caused by my importing some texture hacks that fixes UE4 games.

**Comment by Ant-V** [comment]
I'm in no way an expert when it comes to graphics, but the more I look at it, the more this feels like some sort of reflection issue.

**Comment by Ant-V** [comment]
Issue persists in 22.1.1-2

**Comment by Gcenx** [comment]
> Issue persists in 22.1.1-2

Like I’d mentioned above I don’t believe this has anything todo with wine but MoltenVK

Try swapping out the copy of libMoltenVK.dylib from inside the wrappers /Frameworks directory with the one from this archive https://github.com/The-Wineskin-Project/MoltenVK/releases/download/v1.2.3/macos-1.2.3.tar.xz

**Comment by Ant-V** [comment]
Yup, indeed this fixes it! I do not know if I should mark this as closed though. Maybe add a note somewhere in the wiki?

**Comment by Gcenx** [comment]
> Yup, indeed this fixes it! I do not know if I should mark this as closed though. Maybe add a note somewhere in the wiki?

Lets leave this open for the moment since it’s not fixed within the master wrapper as you’d just fixed it locally.
